gatsby-node.js

```diff
@@ -1,55 +1,36 @@
-const each = require('lodash/each')
-const Promise = require('bluebird')
-const path = require('path')
+const _ = require('lodash')
```

Comment:
Are you using lodash anywhere here? If not, let's remove the import

Reply:
Yes right, I am not using it because we do not have the plugin installed. Do you want me to install lodash or we just do not need it ?